packages/react-notion-x/src/third-party/react-use.ts

```diff
@@ -211,3 +211,19 @@ export const useLocalStorage = <T>(
 
   return [state, set, remove]
 }
+
+// Style mismatches between server rendering and client hydration can act
+// unpredictably. Style that depends on the client state should use this hook
+// can prevent the unpredictable behavior. More details here:
```

Comment:
```suggestion
// unpredictably. Styles that depends on the client state should use this hook
// to prevent the unpredictable behavior. More details here:
```